### DISCUSSION TITLE
If the uploaded file name is in Chinese, the following error message should be displayed:

### DISCUSSION THREAD

**Discussion by huicewang** [open-webui/open-webui#1389] [General]
If the uploaded file name is in Chinese, the following error message should be displayed:
![image](https://github.com/open-webui/open-webui/assets/19237481/b6702336-86fc-4290-8fe3-c417ca430862)
 Why not support Chinese Filename?

**Comment by tjbck** [comment]
Could you share the problematic file with us?
